### ISSUE TITLE
Unable to Sign up for new account or Login via WordPress.com on Pop OS 19.10

### ISSUE THREAD

**Issue by gagan0123** [Automattic/simplenote-electron#1673]
I tried to first login using WordPress.com account, but couldn't, then I tried to create a new account, and even that failed, not using VPN or any Proxy, firewall not blocking any outgoing connections. Sudden spike in CPU usage upon trying to login with WordPress.com account, CPU usage increase dramatically.

### Expected
Upon clicking login with WordPress.com and signing up with account, I should be able to login.

### Observed
Even after clicking autherise, the app just sits there, doing something in background, consuming a lot of CPU and doing nothing in foreground.

### Reproduced
Here's a screencast:
[![Alt text](https://img.youtube.com/vi/j23svqs2XjE/0.jpg)](https://www.youtube.com/watch?v=j23svqs2XjE)

Make|Model|OS Version|App Version
Asus|Rog Strix|Pop OS 19.10|1.8.0

**Comment by belcherj** [comment]
The WordPress.com issue should be resolved in the next version:  https://github.com/Automattic/simplenote-electron/blob/develop/RELEASE-NOTES.txt#L13

I will double-check your account to determine if there are any issues.

**Comment by belcherj** [comment]
I don't see any issues with your account and you should be able to signin.  Can you try the web app? app.simplenote.com

**Comment by gagan0123** [comment]
Thanks for the speedy update on the issue :)
I tested it again with the latest version 1.9.1, still the same issue. 
Here's the screencast with the latest stable version:
[![Alt text](https://img.youtube.com/vi/if0HRBNEcss/0.jpg)](https://www.youtube.com/watch?v=if0HRBNEcss)

Let me know if you need any error logs for the same, or any system stats dump that might be useful.

**Comment by dmsnell** [comment]
@gagan0123 it looks like it's timing-out on the OAuth call and we don't yet have good handling for that. You may be able to look for the network activity in the browser dev tools.

Please don't post your results here (for security's sake) but look for requests to URLs with `oauth` and `WPCC` in them. If one isn't returning or might be returning with an unexpected result then the problem is probably cryptically explained by the error message or lack of response. Note you will need to persist the logs as the signin process moves through a few redirects.

In 1.9.0 we overhauled the existing oAuth flow (WordPress.com signin) but weren't able to close over all the issues - check back in the future for more improvement there.

If none of this makes sense then please feel free to ignore it. Sometimes the signin fails in a way we don't expect. You can contact our support for more assistance.

**Comment by gagan0123** [comment]
@dmsnell I looked a bit deeper and used console in order to try to find out whats going on. On normal chrome browser, I'm able to login and use the simplenote webapp, using my WordPress.com account. In the app, however, the requests are not going through, for some reason. In the network tab of the developer tools, I can see that its trying to send around 5k to 6k requests to the authentication server, every second, which is odd, considering it should at least wait for first request to either timeout or die, before trying another request. Seems thats the reason, some other members who are getting logged out of the accounts, are having their issues, because of excessive API requests.

**Comment by belcherj** [comment]
This should be resolved in 1.10.0 which as luck would have it gets released today!

**Comment by dmsnell** [comment]
Yep! I got that wrong and thought it was already in 1.9.1 but looks like #1627 is part of 1.10.0 - thanks @belcherj 

Also thanks @gagan0123 - it's not your responsibility to debug the app, but it helps us get a better answer to you faster and helps others in the process. 😊

That cycling of requests appears to be related to a change in Electron - we were intercepting `simplenote://` URLs in the response to the OAuth call but since we weren't cancelling those redirects and letting them go through it turned into an infinite loop.

**Comment by gagan0123** [comment]
Can confirm, issue is now resolved upon upgrade to 1.10.0.
Kudos to the development team :+1: 
And thanks for helping out on the issue :)

Though on a side note, I've switched to sublime text for the time being, for some reason, even now with latest version, high CPU usage seems to be another issue. When typing anything, the CPU usage skyrockets, for an instant, it pushes the CPU to 4.5GHz which isn't good for working remotely on a laptop without power supply. I'll raise another issue for this later.
